### ISSUE TITLE
Unable to debug test in VS 2015

### ISSUE THREAD

**Issue by mahendrakariya** [getgauge/gauge-visualstudio#105]
_From @marek2209 on April 19, 2016 10:49_

I have latest version of gauge and i have this problem when i want to start debug in VS  2015

![image](https://cloud.githubusercontent.com/assets/9087374/14636672/20f8caba-062d-11e6-9f1c-c477122ab0b5.png)

_Copied from original issue: getgauge/gauge#345_

**Comment by CL0SeY** [comment]
Debugging is broken for me in VS 2015 Update 2:
```
Test Name:	Vowel counts in multiple word
Test FullName:	[Specification Heading].[Vowel counts in multiple word]
Test Source:	C:\<snip>\specs\example.spec : line -1
Test Outcome:	Failed
Test Duration:	0:00:00

Result Message:	
Object reference not set to an instance of an object.
   at Gauge.VisualStudio.Core.Helpers.DTEHelper.AttachToProcess(Int32 parentProcessId)
   at Gauge.VisualStudio.TestAdapter.GaugeRunner.Run(TestCase testCase, Boolean isBeingDebugged, IFrameworkHandle frameworkHandle)
```

Running tests works fine.

Debugging is working in VS 2013, just not VS 2015.

**Comment by WonkiDonk** [comment]
This issue has been identified as being caused by the same thing that's causing #112. Please watch ticket for #112 for notification of when this issue has been resolved (hopefully in the next 24 hours!)

**Comment by sguptatw** [comment]
Unable to test 'cause the IDE nightly is not compatible with Gauge nightly

**Comment by sriv** [comment]
Should be fixed as part of #112
